source/_docs/migrate.md

```diff
@@ -4,77 +4,324 @@ description: General instructions for preparing and migrating remotely-hosted Dr
+  <li role="presentation"><a href="#d7" aria-controls="d7" role="tab" data-toggle="tab">Drupal 7</a></li>
+  <li role="presentation"><a href="#wp" aria-controls="wp" role="tab" data-toggle="tab">WordPress</a></li>
+</ul>
 
-* Audit your content and code for **hard coded links** pointing to your legacy site.
+<!-- Tab panes -->
+<div class="tab-content">
+  <div role="tabpanel" class="tab-pane active" id="d8">
+  <p>Archives for Drupal 8 sites should include <code>index.php</code> at the code root level, along with the following directories:</p>
+  <pre><code class="nohighlight">
+  ├── core
+  ├── index.php
+  ├── modules
+  ├── profiles
+  ├── sites
+      └── all
+         ├── modules
+         └── themes
+      └── default
+         └── settings.php
+  └── themes
+  </code></pre>
+  </div>
+  <div role="tabpanel" class="tab-pane" id="d7">
+  <p>Archives for Drupal 7 sites should include <code>index.php</code> at the code root level, along with the following directories:</p>
+  <pre><code class="nohighlight">
+  ├── includes
+  ├── index.php
+  ├── misc
+  ├── modules
+  ├── profiles
+  ├── scripts
+  ├── sites
+      └── all
+         ├── modules
+         └── themes
+      └── default
+         └── settings.php
+  └── themes
+  </code></pre>
+  </div>
+  <div role="tabpanel" class="tab-pane" id="wp">
+  <p>Archives for WordPress sites should include <code>index.php</code> at the code root level, along with the following directories:</p>
+  <pre><code class="nohighlight">
+  ├── index.php
+  ├── wp-activate.php
+  ├── wp-config.php
+  ├── wp-comments-post.php
+  ├── wp-blog-header.php
+  ├── wp-admin
+  ├── wp-cron.php
+  ├── wp-load.php
+  ├── wp-links-opml.php
+  ├── wp-includes
+  ├── xmlrpc.php
+  ├── wp-trackback.php
+  ├── wp-signup.php
+  ├── wp-settings.php
+  ├── wp-mail.php
+  ├── wp-login.php
+  ├── wp-content
+      ├── index.php
+      ├── mu-plugins
+      ├── themes
+      ├── plugins
 
-* **Review and archive your legacy site's logs.** The root causes for errors or slow transactions occurring on your legacy site will probably be migrated over, so it would be a good time to resolve them prior to the migration. Slow MySQL queries should be targeted for optimization, as a distributed infrastructure adds a small amount of latency on each transaction.
+  </code></pre>
+  </div>
+</div>
 
-* **Review your configuration files** (my.conf, php.ini, .htaccess, etc). Any custom settings will need to be reviewed, as Pantheon provides platform-wide configuration. If, for example, the site allows very long timeouts, hides PHP errors, or changes how mySQL transactions are handled; these changes may not exist after migration, and may lead to unexpected results. Also, NGINX doesn't process the Apache-specific .htaccess file, so any redirects or custom logic within those files will need to be reconsidered.
 
-## Troubleshooting
+### Could not import database, unable to locate a database dump. ###
```

Comment:
This is one of three sections in `## Troubleshooting` I think we should expand, as they seem to only restate the error message.